app/api_client.py

```
@@ -88,3 +88,18 @@ def preprocess_source(self, data, source):
     @api_error_handler
     def get_queue(self):
         return requests.get(f"{self.base_url}/transcription/queue/")
+
+    @api_error_handler
+    def add_translation_to_queue(self, data, source):
+        with open(source, "rb") as f:
+            files = {"source_file": (source, f, "text/plain")}
+            return requests.post(
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)